src/features/no_recommended.json

```
@@ -13,6 +13,11 @@
       "label": "Hide experimental types of recommended posts",
       "default": true
     },
+    "hide_recommended_community_posts": {
+      "type": "checkbox",
+      "label": "Hide posts from non-followed communities in For You",
```

The official terminology decrees that a community cannot be followed, only joined.

```suggestion
      "label": "Hide posts from non-joined communities in For You",
```

---

`un-joined`, `unjoined`, no... `communities you haven't joined` is long enough to wrap in an ugly way... yeah, sure, `non-joined`, good enough